salt/elife-bot/processes.sls

```diff
@@ -44,38 +44,16 @@ elife-bot-processes-start:
 
 {% else %}
 
+# 16.04+
 
-
-{% set controller = "elife-bot-processes" %}
-
-{{ controller }}-script:
+{% for process in pillar.elife.multiservice.services %}
+elife-bot-{{ process }}-service:
     file.managed:
-        - name: /opt/{{ controller }}.sh
-        - source: salt://elife/templates/systemd-multiple-processes-parallel.sh
+        - name: /lib/systemd/system/{{ process }}@.service
+        - source: salt://elife-bot/templates/lib-systemd-system-botprocess@.service
         - template: jinja
         - context:
-            processes: {{ processes }}
-            # ResizeImages and other activities run for a very long time
-            timeout: 300
-
-{{ controller }}-service:
-    file.managed:
-        - name: /lib/systemd/system/{{ controller }}.service
-        - source: salt://elife-bot/config/lib-systemd-system-{{ controller }}.service
-
-    service.running:
-        - name: {{ controller }}
-        - require:
-            - cmd: register-swf
```

Comment:
I think this is an important requisite. Adding it back in.

The `watch` requisite doesn't matter because the service is restarted regardless on every highstate

The `listen` requisites also don't matter for the same above reason.